03week/towersOfHanoi.js

```diff
@@ -7,38 +7,93 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
-let stacks = {
+//This will be the array we need to access to move piece by piece.
+
+const stacks = {
   a: [4, 3, 2, 1],
   b: [],
   c: []
 };
 
+
+
 function printStacks() {
   console.log("a: " + stacks.a);
   console.log("b: " + stacks.b);
   console.log("c: " + stacks.c);
 }
 
-function movePiece() {
+//I think the logic on the functions would be better with 
+//a const function. So I need to capture the value in the array
+//the array seems to be inside an object.
+// I would need a function that will take a value from one aray
+// and place them the other arrays with in stacks
+
+const movePiece= (startStack, endStack)=> {
   // Your code here
+  // I am creating a variable with the user input values
 
-}
+  const startMove = stacks[startStack]
+  const endMove = stacks[endStack];
 
-function isLegal() {
-  // Your code here
+  //Add a function that take user input with legal move rules
+  //all together shoud take one value from an array an move it
+  // to another array with out the vaule being greater than the last value
 
-}
+  if (isLegal(startMove[startMove.length-1], endMove[endMove.length-1])){
```

Comment:
your movePiece function shouldn't be calling your isLegal function, this leads to messy spaghetti code, your parent Towers of Hanoi function should call your isLegal function and depending on the return, call the movePiece function